www/lendingbot.js

```diff
@@ -246,6 +258,8 @@ function Timespan(name, multiplier) {
             }
             if (currency == "BTC") {
                 return displayUnit.formatValue(earnings) + " <span class=" + currencyClass + ">" + displayUnit.name + "</span> / " + name + "<br/>"
+            } else if (currency == "USD" || currency == "USDT") {
+                return printFloat(earnings, 2) + " <span class=" + currencyClass + ">" + currency + "</span> / "+  name + "<br/>";
```

Comment:
@ckcollab "EUR" for example and actually any currency supported by https://blockchain.info/api/exchange_rates_api

I'm thinking about using the prettyFloat as it will provide an accuracy of 2 decimals for any coin and I think that should be good enough, no? or should we make the accuracy configurable? 

I'm open for suggestions.